packages/utilities/psammead-locales/moment/ar.test.js

```diff
@@ -0,0 +1,259 @@
+/* eslint-disable */
+
+import moment from 'moment';
+import './ar';
+
+moment.locale('ar');
+
+// This asset overrides the gunit assertion done in the moment codebase.
+// Format and styling of this file has been keep consistent with the official moment tests.
+// An example of these tests can be seen at https://github.com/moment/moment/blob/develop/src/test/locale/en-gb.js
+const assert = { equal: (val1, val2) => expect(val1).toEqual(val2) };
```

Comment:
Why is this needed? I know it's in `yo.js` but I want to be sure it is required rather than just blindly copying it, and if it is needed I think the comment should explain why.

Reply:
if not used, the tests fails with this error `assert.equal is not a function`

Reply:
Clarified with @aacn500 that this was done originally in order to make it potentially easier to contribute the tests back to Moment.js without rewriting them.